frontend/sozisel-app/src/components/SessionResultScreen/SessionResultEvents/EventResultDetails/Modules/QuizResultDetails/QuizResultDetails.scss

```diff
@@ -0,0 +1,89 @@
+@import "../../../../../../styles/styles";
+
+
+.QuizResultDetails {
+    height: calc(100% - #{$navbar-height} - #{6 * $spacing-09} - #{$spacing-08});
+    width: calc(100% - 2*#{$spacing-09});
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding: $spacing-09;
+    @include poppins-text;
+
+    .quizSummary {
+        flex: 1;
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        margin-right: $spacing-07;
+        padding: $spacing-06;
+    }
+
+    .quizResultView {
+        flex: 3;
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        padding: $spacing-06;
+    }
+
+    .headerWithIcon {
+        display: flex;
+        flex-direction: row;
+        justify-content: start;
+        align-items: center;
+        margin-bottom: $spacing-07;
+    }
+
+    .header{
+        color: $text-04;
+        font-size: 32px;
```

Comment:
If you want to use pixels I guess it's okay as the project is not that much significant but at least be consistent across the file, either all pixels or all spacings